app/src/main/java/com/umc/edison/presentation/login/IdentityTestViewModel.kt

```diff
@@ -73,94 +63,99 @@ class IdentityTestViewModel @Inject constructor(
                 category = identityCategory,
             ),
             onSuccess = { identity ->
-                _uiState.update {
-                    it.copy(identity = identity.toPresentation())
+                val model = identity.toPresentation()
+                _uiState.update { state ->
+                    state.copy(
+                        identities = state.identities.toMutableMap().apply {
+                            // 카테고리별로 IdentityModel 저장
+                            put(identityCategory, model)
+                        }
+                    )
```

Comment:
mutableMap으로 컨버팅 할 필요 없이 아래처럼 사용도 가능해요

```suggestion
                    state.copy(
                        identities = state.identities + (identityCategory to model)
                    )
```